<? 
include 'includes.php';
$title = 'Links';
$curr = 'links';
phead($title);
ptabs($curr);
?>
<div class="content">
  <div class="linkgroup">
      <h3>Related Journals</h3>
    <ul>
   <li><a href="http://pubs.acs.org/journals/bichaw/">
   Biochemistry</a></li>
   <li><a href="http://www.cell.com/">
   Cell</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.immunity.com/">
   Immunity</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.cellmetabolism.org/">
   Metabolism</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.molecule.org/">
   Molecular</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.neuron.org">
   Neuron</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://genomebiology.com/">
   Genome Biology</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://www.genome.org/">
   Genome Research</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://www.jbc.org">
   Journal of Biology Chemistry</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://jbr.sagepub.com/">
   Journal of Biological Rhythms</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://www.jcb.org">
   Journal of Cell Biology</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://mcb.asm.org/">
   Molecular and Cellular Biology</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://www.nature.com/nature/index.html">
   Nature</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/biotech/index.html">
   Biotechnology</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/emboj/index.html">
   EMBO</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/genetics/index.html">
   Genetics</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/molcellbio/index.html">
   Molecular cell biology</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/medicalresearch/index.html">
   Medical Research</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/micro/index.html">
   Microbiology</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/neurosci/index.html">
   Neuroscience</a></li>
   <li style="text-indent:15px;">
   <a target="_blank" rel="nofollow" href="http://www.nature.com/pharma/index.html">
   Pharmacology</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://www.pnas.org" title="Proceedings of the National Academy of Sciences">
   PNAS</a></li>
   <li>
   <a target="_blank" rel="nofollow" href="http://www.sciencemag.org/">
   Science Magazine</a></li>
  </ul>
  </div>
 
  <div class='linkgroup'>
      <h3>Online Databases</h3>
      <ul>
   <li><a target="_blank" rel="nofollow" href="http://www.ncbi.nlm.nih.gov/BLAST/">
   BLAST</a>
    <a href="#" class="trigger"></a>
    <code><br />The Basic Local Alignment Search 
   Tool (BLAST) finds regions of local similarity 
   between sequences. The program compares nucleotide 
   or protein sequences to sequence databases and 
   calculates the statistical significance of matches. 
   BLAST can be used to infer functional and 
   evolutionary relationships between sequences as well 
   as help identify members of gene families. <br /><br />
    </code><div class="show" />
   </li>
             
    <li>
    <a target="_blank" rel="nofollow" href="http://expression.gnf.org/circadian">
    
    Circadian Gene Expression Database</a></li>
    <li>
     <a target="_blank" rel="nofollow" href="http://david.abcc.ncifcrf.gov/tools.jsp">
     DAVID</a>
     <a href="#" class="trigger"></a>
     <code><br />
      The Database for Annotation, 
     Visualization and Integrated Discovery 
     (DAVID) 2006 is an expanded version of 
     original web-accessible program. DAVID 
     provides a comprehensive set of functional 
     annotation tools for investigators to 
     understand biological meaning behind large 
     list of genes.<br /><br /> 
     </code><div class="show" />
    </li>
    
    <li>
     <a target="_blank" rel="nofollow" href="http://www.ensembl.org/index.html">
      Ensemble Genome Browser</a>
    </li>
    <li>
     <a target="_blank" rel="nofollow" href="http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar">
     Entrez</a>
     <a href="#" class="trigger"></a>
     <code><br />
      Entrez is a powerful federated search 
     engine, or web portal that allows users to 
     search a multiple of discrete health 
     sciences databases, maintained by the 
     National Center for Biotechnology 
     Information with a single query string and 
     user interface.   
      <br /><br /> 
     </code><div class="show" />
    </li>

    <li>
    <a target="_blank" rel="nofollow" href="http://www.ncbi.nlm.nih.gov/">
    NCBI Genbank</a></li>
    <li>
    <a target="_blank" rel="nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed">
    NCBI Pubmed</a></li>
    <li>
    <a target="_blank" rel="nofollow" href="http://www.rcsb.org/pdb/home/home.do">
    PDB</a></li>
    <li>
    <a target="_blank" rel="nofollow" href="http://genome.ucsc.edu/cgi-bin/hgGateway">
    UCSC Genome Browser</a></li>
    <li>
     <a target="_blank" rel="nofollow" href="http://www.expasy.org/sprot/">
     UniProtKB/Swiss-Prot</a>
     <a href="#" class="trigger"></a>
     <code><br />
      The UniProtKB/Swiss-Prot Protein 
     Knowledgebase is a curated protein sequence 
     database that provides a high level of 
     annotation, a minimal level of redundancy 
     and a high level of integration with other 
     databases. Together with UniProtKB/TrEMBL, 
     it constitutes the UniProt Knowledgebase, 
     one component of the Universal Protein 
     Resource (UniProt), a one-stop shop allowing 
     easy access to all publicly available 
     information about protein sequences.<br /><br />
     </code><div class="show" />
    </li>     
    <li>
     <a target="_blank" rel="nofollow" href="http://symatlas.gnf.org/SymAtlas/">     
     Symatlas</a><a href="#" class="trigger"></a>
     <code><br />
      SymAtlas is a web-application for 
     publishing experimental gene 
     functionalization datasets integrated with a 
     flexibly searchable gene-centric database of 
     public and proprietary annotations.<br /><br />    
     </code><div class="show" />
    </li>
   </ul>
  </div>
  <div class="linkgroup">
      <h3>Bioinformatics Software</h3>
    <ul>
			<li>
				<a target="_blank" rel="nofollow" href="http://hmmer.janelia.org/">
				HMMER</a>
				<a href="#" class="trigger"></a>
				<code><br />
					HMMER is an implementation of profile 
				hidden Markov model (HMM) methods for 
				sensitive database searches using multiple 
				sequence alignments as queries. 
					<br /><br />
					HMMER takes a multiple sequence 
				alignment as input. It can then build a 
				statistical model called a &quot;hidden Markov 
				model&quot; which can be used as a query into a 
				sequence database to find (and/or align) 
				additional homologues of the sequence 
				family.<br /><br />
				</code><div class="show" />
			</li>
			
			<li>
				<a target="_blank" rel="nofollow" href="http://www.r-project.org/">
				R</a>
				<a href="#" class="trigger"></a>
				<code><br />							
					R is a language and environment for 
				statistical computing and graphics, 
				itprovides a wide variety of statistical 
				(linear and nonlinear modelling, classical 
				statistical tests, time-series analysis, 
				classification, clustering, ...) and 
				graphical techniques, and is highly 
				extensible.<br /><br />
				</code><div class="show" />
			</li>

			
			<li>
			<a target="_blank" rel="nofollow" href="http://www.bioconductor.org">
			Bioconductor</a></li>
			<li>
				<a target="_blank" rel="nofollow" href="http://genome.ucsc.edu/cgi-bin/hgBlat">
				BLAT</a>
				<a href="#" class="trigger"></a>
				<code><br />
						A tool performs rapid mRNA/DNA and 
				cross-species protein alignments.<br /><br />
				</code><div class="show" /></li>
			<li>
				<a target="_blank" rel="nofollow" href="http://rana.lbl.gov/EisenSoftware.htm">
				Cluster and Treeview</a>
				<a href="#" class="trigger"></a>
				<code><br />
					Cluster and TreeView are an integrated 
				pair of programs for analyzing and 
				visualizing the results of complex 
				microarray experiments. <br /><br />
				</code><div class="show" />
			</li>

			<li>
				<a target="_blank" rel="nofollow" href="http://genes.mit.edu/GENSCAN.html">
				Genscan</a>
				<a href="#" class="trigger"></a>
				<code><br />
					GENSCAN is a program designed to predict 
				complete gene structures, including exons, 
				introns, promoter and poly-adenylation 
				signals, in genomic sequences. It differs 
				from the majority of existing gene finding 
				algorithms in that it allows for partial 
				genes as well as complete genes and for the 
				occurrence of multiple genes in a single 
				sequence, on either or both DNA strands. 
				Program versions suitable for vertebrate, 
				nematode (experimental), maize and 
				Arabidopsis sequences are currently 
				available. The vertebrate version also works 
				fairly well for Drosophila sequences. 
				Sequences can be submitted on a web-based 
				form at this site. <br /><br />
				</code><div class="show" />
			</li>			


			<li>
			<a target="_blank" rel="nofollow" href="http://www.ruby-lang.org/en/">
			Ruby</a></li>
			<li>
			<a target="_blank" rel="nofollow" href="http://bioruby.org/">
			BioRuby</a></li>
			<li>
			<a target="_blank" rel="nofollow" href="http://www.perl.org/">
			Perl</a></li>
			<li>
			<a target="_blank" rel="nofollow" href="http://www.bioperl.org/wiki/Main_Page">
			BioPerl</a></li>
		</ul>
  </div>
</div>

<? pfoot(); ?>